googlesheets/plugin.go

```diff
@@ -49,11 +50,24 @@ func PluginTables(ctx context.Context, p *plugin.TableMapData) (map[string]*plug
 		return tables, nil
 	}
 
-	// Return all valid sheets
+	// Retrieve all valid sheets.
+	// If no sheet is specified in the connection config and Steampipe is started, no table map will be added for any of the sheets.
+
+	// Wildcard:
+	//
+	// sheets = ["*"]
+	// Table mapping will be performed for all available sheets.
+
+	// Partial wildcard:
+	//
+	// sheets = ["Student*", "Work*"]
+	// Table mapping will be performed for the available sheets matching the patterns provided in the SPC file.
```

Comment:
```suggestion
	// Retrieve all valid sheets
	// If no sheets are specified in the config arg, no dynamic tables will be created
```

I don't think we need the full examples here in the comments since we already have them in the arg description